web/src/admin/ProjectsAdmin.tsx

```diff
@@ -59,7 +62,7 @@ const ProjectsAdmin = (props: any) => {
                 // label={metaKey}
                 defaultValue={projectMeta[metaKey]}
                 // onChange={(e) => setProjectMetaState({ [project.name!]: { meta: { ...projStateMeta, [metaKey]: e.target.value } } })}
-                onBlur={(e) => {
+                onBlur={(e: any) => {
```

Comment:
Try to avoid `any` types. I think it should be `e: React.FocusEvent<HTMLInputElement>`

Reply:
Amazing, thank you!